aws-go-console-slack-notification/main.go

```diff
@@ -19,15 +15,14 @@ import (
 
 func main() {
 	pulumi.Run(func(ctx *pulumi.Context) error {
-
 		config := config.New(ctx, "")
 
 		var regions []string
 		config.GetObject("regions", &regions)
 
 		if len(regions) == 0 {
 			var err error
-			regions, err = getRegions()
+			regions, err = getRegions(ctx)
```

Comment:
Using built-ins from Pulumi makes authentication story much easier, for example AWS_PROFILE just works.